config/rbac/role.yaml

```diff
@@ -6,6 +6,16 @@ metadata:
   creationTimestamp: null
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
This is not at role level because validatingwebhookconfigurations is not namespaced, right?

Reply:
yes @fedepaol , this is in the cluster-role.